### ISSUE TITLE
Tutorials icon images ratio not respected

### ISSUE THREAD

**Issue by VictorLamoine** [PointCloudLibrary/pcl#1329]
Online
===
![online](https://cloud.githubusercontent.com/assets/5566160/9821181/fb38d7c4-58b8-11e5-94e9-2bc0ed897440.png)
Local
===
![local](https://cloud.githubusercontent.com/assets/5566160/9821183/fc2f0aa4-58b8-11e5-8bbb-4477dbb652b3.png)

**Comment by taketwo** [comment]
Online version looks good from Firefox though...

**Comment by VictorLamoine** [comment]
That's right, it's fine in Firefox, not in Chromium, I tried in private mode with Chromium and still get the issue (I'm unsure if private mode uses user config)

It also happens on my Android phone so I guess it's not my user configuration making the images looks deformed.

**Comment by taketwo** [comment]
Sure, it's not a problem with your configuration, I observe the same in my default browser (Chromium). Just wanted to throw in additional piece of information.

**Comment by taketwo** [comment]
The problem comes from this `colgroup` tag, which _Sphinx_ inserts in every table:

```html
<colgroup>
  <col width="6%">
  <col width="94%">
</colgroup>
```

Here is a [relevant discussion](http://stackoverflow.com/questions/21027105/how-can-i-have-sphinx-tables-fit-to-width), though no good solution.
